base_layer/common_types/src/wallet_types.rs

```diff
@@ -60,7 +60,11 @@ pub struct ProvidedKeysWallet {
 impl Display for ProvidedKeysWallet {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         write!(f, "public spend key {}", self.public_spend_key)?;
-        write!(f, "public view key{}", PublicKey::from_secret_key(&self.view_key))?;
+        write!(
```

Comment:
Actually a problem in the original code. We should now allow heavy code in a `to_string()` method. This should be stored rather

Reply:
yeah, I removed it from the display as you can do nothing with the public view key, so why display it in the first place